lib/hpke/include/hpke/certificate.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <bytes/bytes.h>
+#include <hpke/signature.h>
+
+#include <memory>
+
+using namespace bytes_ns;
+
+namespace hpke {
+
+struct Certificate
+{
+private:
```

Comment:
Nit: Move this to the bottom of the struct, and then you won't need the `public:` annotation.

Reply:
leaving it as is to have compiler construct things in the order we need here.